_posts/2019-02-01-dynamic-spawn-scheduler.md

```diff
@@ -71,14 +77,16 @@ iOS | iMac Pro 2018, 18-core | 34s | 414s ↑ | 32s ≈
 iOS | Mac Pro 2013, 6-core | 39s | 414s ↑ | 36s ≈
 iOS | MacBook Pro 2015, 4-core | 39s | 450s ↑ | 36s ≈
 
-As you can see, the remote-only configuration gave us slightly better results than the local-only configuration for clean builds, but the remote-only configuration became unacceptably slow for incremental builds. As expected, though, dynamic execution wins heads-down in all cases: clean builds are many times faster than they are with local or remote execution alone, and incremental builds are essentially the same.
+As you can see, the remote-only configuration gave us better results than the local-only configuration for clean builds, but the remote-only configuration became unacceptably slow for incremental builds. As expected, though, dynamic execution won heads-down in all cases: clean builds were many times faster than they are with local or remote execution alone, and incremental builds were essentially the same.
```

Comment:
Neat! These are pretty sweet results, nice work :).

Reply:
Oh, one more point: if you wait till EoW and then re-test, the remote times should get better across the board: iirc one of the issues with these builds was that they have lots of inputs per action, and our current release has performance improvements landing targeting that use-case.

I wouldn't block publishing before that though.

Reply:
Cool. @jin can retest Android when he's back, which should be EoW as well :) iOS is purely internal so your fixes wouldn't fix that :-/

Reply:
Yeah, sorry, I meant to say "RBE times" vs all remote. And to be clear, I don't know what the magnitude of the impact will be - certainly not enough to make RBE competitive with local for incremental android builds I don't think, but hopefully not as painfully far :).